compiler/rustc_codegen_ssa/src/glue.rs

```diff
@@ -46,7 +46,7 @@ pub fn size_and_align_of_dst<'a, 'tcx, Bx: BuilderMethods<'a, 'tcx>>(
                 // NOTE: ideally, we want the effects of both `unchecked_smul` and `unchecked_umul`
                 // (resulting in `mul nsw nuw` in LLVM IR), since we know that the multiplication
                 // cannot signed wrap, and that both operands are non-negative. But at the time of writing,
-                // `BuilderMethods` can't do this, and it doesn't seem to enable any further optimizations.
+                // the `LLVM-C` binding can't do this, and it doesn't seem to enable any further optimizations.
```

Comment:
Just to keep someone else from going "well adding it to our trait sounds easy" -- the actual hard part is adding it to

https://github.com/llvm/llvm-project/blob/681d5eecf7b32748763e5a8f56f7c334d8d6901e/llvm/include/llvm-c/Core.h#L3873-L3876